BREAKING.md

```diff
@@ -73,3 +73,301 @@ func createSQSAPI(endpoint string) (*sqs.Client, error) {
 ```
 
 > A more detailed documentation on migrating can be found [here](https://aws.github.io/aws-sdk-go-v2/docs/migrating).
+
+## v1.0.0
```

Comment:
We are not quite there yet for v1.0.0. A lot more things need to happen for v1.
If we look at our releases this will be more likev0.74.0.
The examples given seem a lot.
We should just give a simple before and after.
WDYT?

Reply:
Yeah, I think it makes more sense.